### PR TITLE
issue/1029 Narrative ES6 and text below image

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes https://github.com/adaptlearning/adapt_framework/issues/1029
fixes https://github.com/adaptlearning/adapt_framework/issues/1596

### Added
* `_isTextBelowImage` 
* `_isMobileTextBelowImage`
* Swipeleft and swiperight navigation behaviour

### Changed
* Converted to ES6 modules

Requires https://github.com/adaptlearning/adapt_framework/pull/3179 for swiping support

### CODE REVIEW

**Review by guywillis — APPROVED** [review]
👀

File: README.md [hunk 1]
```diff
@@ -48,6 +48,10 @@ The attributes listed below are used in *components.json* to configure **Narrati
 
 **mobileInstruction** (string): This is optional instruction text that will be shown when viewed on mobile. It may be used to guide the learner’s interaction with the component.
 
+**\_isTextBelowImage** (boolean): If enabled, the text area drops below the image instead of being aside it.
```
Comment (cahirodoherty-learningpool) [inline README.md:51]: Minor: ~aside~ 'beside'

**Review by cahirodoherty-learningpool — CHANGES_REQUESTED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

**Pull request merged**